libs/pageserver_api/src/models.rs

```diff
@@ -641,6 +643,12 @@ pub struct TimelineGcRequest {
     pub gc_horizon: Option<u64>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct WalRedoManagerStatus {
+    pub last_redo_at: Option<chrono::DateTime<chrono::Utc>>,
```

Comment:
I do not if this has anything sensible jsonification by default (I hope rfc3339) but let's just merge it and fix later.

Reply:
Same concern, but also okay with fixing it up later.  Is there an echo in here? ;-)